packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx

```diff
@@ -143,24 +135,29 @@ const Group: React.FC<{
   onSelect: (pageId: PageId) => void;
 }> = ({ pages, depth, partial, selectedPageId, onSelect }) => {
   const [isOpened, setIsOpened] = useState<boolean>(true);
-  const handleClick = useCallback<TextOnButtonProps['onClick']>(() => {
+  function handleClick() {
```

Comment:
Will we go with `function` , not arrow function ?

Reply:
I just prefer `function`. 
If you want, I will change to `arrow function`.

Reply:
Both will do, but i just want go with one way!